doc/source/how-to/documenting.rst

```diff
@@ -913,4 +913,114 @@ files for inclusion in the documentation.
 | examples/02-HFSS/HFSS_Spiral.py                 | examples-design-and-simulation-of-spiral-inductors.py |
 +-------------------------------------------------+-------------------------------------------------------+
 
-If you have questions about SEO, email the `PyAnsy Core team <pyansys_core_email_>`_.
+If you have questions about SEO, email the `PyAnsys Core team <pyansys_core_email_>`_.
+
+Facilitating contributions in the form of examples
+--------------------------------------------------
+
+A common challenge in libraries is the lack of examples. Examples are
+critical for users to understand how to use the library and present new
+functionality. They also serve as a mechanism to educate users on best practices.
+
+To encourage contributions of examples, it is important to provide a clear
+structure for contributors to follow. The build process should also be straightforward.
+
+Two best practices for examples are:
+
+- **Providing a template**: Ensures that contributors follow a consistent structure and
+  simplifies the process of getting started. The template should include all necessary
+  information, such as the name of the example, a description, and the code itself.
+- **Verifying the example**: Ensures that the example works as expected and helps catch
+  errors before submission. The verification process should involve running the example
+  and checking the output. If the example fails, it should be easy to identify and fix the issue.
+- **Making the example usable in multiple contexts**: Ensures that the example can be
+  usable by a user reading the documentation, an application engineer using the library
+  through a notebook, or a developer including the example to the documentation.
+
+Providing a template
+~~~~~~~~~~~~~~~~~~~~
+
+Here are examples of how to promote the use of templates for examples:
+
+- `PyAnsys Geometry examples template <PyAnsys_geometry_examples_template_>`_
+- `PyPrimeMesh examples template <PyPrimemesh_examples_template_>`_
+
+These templates are also referenced in the contributing guidelines:
+
+- `PyAnsys Geometry example contribution guidelines <PyAnsys_geometry_examples_contribution_>`_
+
+Verifying examples
+~~~~~~~~~~~~~~~~~~
+
+The verification process depends on the library and the tools used for generating the examples.
+Within the PyAnsys ecosystem, Sphinx Gallery and NBSphinx are commonly used to generate examples,
+with slightly different build processes for each library.
+
+The following approach can be applied across libraries:
+
+- Create a ``Makefile`` target that runs a specific example.
+
+This target is used to execute the example and check the output. If the example fails,
+it should be straightforward to identify and resolve the issue. For instance, PyAnsys Geometry
+includes a ``make`` target for this purpose:
```

Comment:
I was thinking that we could also specify that the documentation will **only** renders the specific example which results in a faster output. It can be very useful for long documentation runs or for CI/CD pipelines running on a single VM.